ci/run.sh

```diff
@@ -95,12 +96,14 @@ fi
 
 case "$TARGET" in
   *-apple-ios)
-    cargo rustc --manifest-path libc-test/Cargo.toml --target $TARGET -- \
-        -C link-args=-mios-simulator-version-min=7.0
+    cargo rustc --manifest-path libc-test/Cargo.toml --target $TARGET \
+        --test main -- -C link-args=-mios-simulator-version-min=7.0
+    cargo rustc --manifest-path libc-test/Cargo.toml --target $TARGET \
+        --test linux-fcntl -- -C link-args=-mios-simulator-version-min=7.0
```

Comment:
It may be worth testing out `RUSTFLAGS` nowadays. I think at the time this code was written `RUSTFLAGS` didn't exist but nowadays I think it'd suffice for this, allowing us to basically just use `cargo build --tests`